python/spacewalk/satellite_tools/repo_plugins/__init__.py

```diff
@@ -115,6 +115,14 @@ def set_checksum(self, checksum_type_in=None, checksum_in=None):
             if not((checksum_type_in in self.checksums) and (self.checksums[checksum_type_in] == checksum_in)):
                 self.checksums[checksum_type_in] = checksum_in
 
+    def get_nvra_tuple(self) -> tuple:
+        """returns a tuple of name, version, release, arch"""
+        return (self.name, self.version, self.release, self.arch)
```

Comment:
A bit above we have `getNRA`, `getNVREA`, and `getNEVRA`. Do we really need `get_nvra_tuple` as well?

The only caller is `is_metadata_valid` which has direct access to instance members, is this level of indirection needed?